### ISSUE TITLE
Відредагувати український переклад

### ISSUE THREAD

**Issue by konfuciusu** [hurtom/toloka#1]
https://crowdin.com/project/torrentpier/uk#

**Comment by Kobzar09** [comment]
Потрібно просто перечитати переклад на наявність помилок?
Які стрічки вже перевірені? 
Ті що із галочкою чи із квадратиком?

**Comment by konfuciusu** [comment]
Перевірені ті, що з галочкою

Основний текст тут: https://crowdin.com/translate/torrentpier/93/en-uk

**Comment by Kobzar09** [comment]
Що потрібно натиснути, щоб затвердити переклад?

**Comment by konfuciusu** [comment]
Можна тільки перекладати і голосувати

Пропоную написати https://github.com/Exile37, щоб він зробив вас пруфрідером, тоді у вас буде можливість також затверджувати

**Comment by Kobzar09** [comment]
Написав йому, побачимо що він скаже. Не знаю чи там потрібен якийсь мінімальний термін перебування на сайті чи показник перекладу для отримання цього статусу.

**Comment by Kobzar09** [comment]
Додав той чел до пруфрідерів, зараз перевіряю, правда я не впевнений на 100 % у всіх перекладах, що там є. Не завадило погляду зі сторони.

**Comment by viksok** [comment]
прогорнув пару сторінок стрігнів на переклад - в деяких багато неточостей.
також нема контексту в якому використовується стрінг - звідки можна бути певним що переклад має бути "Нове" а не "Новий" чи "Нова" і тому подібне.

**Comment by konfuciusu** [comment]
Для контексту найкраще запустити сайт, для цього треба тільки git і docker. Якщо треба, я допоможу, пишіть в https://gitter.im/hurtom/toloka

**Comment by Kobzar09** [comment]
@viksok Ви допоможете з перекладом?
Мені б не завадив ще один варінт перекладу для підтвердження.
Зараз спробую поставити гіт+докер, щоб дивитися як воно виглядає.

Буду радий бачити Вас в команді перекладу =)

**Comment by Kobzar09** [comment]
На сайті перекладу двигуна ТоррентПаєр є ще розділи:
 - Обговорення - https://crowdin.com/project/torrentpier/discussions/2/0/home 
 - Активність - https://crowdin.com/project/torrentpier/activity_stream
Не забувайте відвідувати =)

**Comment by Kobzar09** [comment]
Звертання "Ви", "Вас" і т.п писати з великої літери

**Comment by viksok** [comment]
@Kobzar09 так, можу допомогти. Файтично вже почав переглядати і редагувати.
Стараюсь теж лишати коментарі до суперечливих перекладів.

@konfuciusu де лежать образи докера? Хочу поставити локально.

**Comment by konfuciusu** [comment]
@viksok в [readme](https://github.com/hurtom/toloka#%D0%9F%D0%BE%D1%87%D0%B0%D1%82%D0%BE%D0%BA) описано, як запустити образи. Встановлення docker'а описано тут: https://github.com/hurtom/toloka/issues/25

**Comment by Kobzar09** [comment]
Як правильно перекладається на трекері **announce**?

Доречі потрібно створити словник, щоб однакові слова перекладали всі редактори однаково.

Щоб не брали різні слова для перекладу одного і того д терміну.
Наприклад, заголовок Attachments - Вкладення, а кнопка буде - Attach - прикріпити, або додати.

**Comment by konfuciusu** [comment]
Такі дуже технічні слова, як announce, можна було би і залишити. Скоріш за все воно згадується тільки в адмін-панелі.

Щодо словника - згоден, пропоную створити для цього ukrainian.md в теці, де readme.md

**Comment by Kobzar09** [comment]
Як створити той файл?

**Comment by sergii-zhuravel** [comment]
Як забрати переклад з сайту перекладів для тестування? Багато слів та виразів будуть більш зрозумілими якщо дивитися в "бойових" умовах)

**Comment by Kobzar09** [comment]
Як я розумію, переклад можна завантажити після повного перекладу та затвердження, потім коли вже весь переклад буде завершено, тоді перейти до етапу тестування.

**Comment by konfuciusu** [comment]
@Kobzar09 
>Як створити той файл?

[https://github.com/hurtom/toloka#vi-Створення зміни](https://github.com/hurtom/toloka#vi-%D0%A1%D1%82%D0%B2%D0%BE%D1%80%D0%B5%D0%BD%D0%BD%D1%8F-%D0%B7%D0%BC%D1%96%D0%BD%D0%B8)

**Comment by konfuciusu** [comment]
Exile37 оновив український переклад, тепер його можна бачити вживу. https://gitter.im/hurtom/toloka?at=5942ec76f31c8ced0c3d04cd

**Comment by Kobzar09** [comment]
Переклад завершено, якщо будуть якісь правки - пишіть :)

**Comment by yukoff** [comment]
@Kobzar09 Треба б наявні правки передивитись - я пройшовся по тим місцям, які помітив у докері, але поки що продивився не все. Там і до мене залишали зауваження - треба їх продивитись. Пруфрідер, я так розумію, наразі один?

**Comment by Kobzar09** [comment]
Так, поки в мене відпустка зроблю. Але потім мене не буде довгий час. 
Тому щось думайте. Але зараз поставив знову вінду, тому докер знову ставити не буду.
Зможу тільки через браузер щось змінити.
